polaris-react/src/components/Listbox/Listbox.stories.tsx

```diff
@@ -9,13 +9,72 @@ import {
   Listbox,
   LegacyStack,
   AutoSelection,
+  VerticalStack,
+  Text,
+  Box,
 } from '@shopify/polaris';
 import {CirclePlusMinor, SearchMinor} from '@shopify/polaris-icons';
 
+import {useFeatures} from '../../utilities/features';
```

Comment:
you can also use the storybook global context value if you want! But if this works thats ok w me :)